### DISCUSSION TITLE
Looking good!

### DISCUSSION THREAD

**Discussion by EtherGhost** [wieslawsoltes/DataBox#11] [General]
Looks really good!
What is the long-term plan with it? Will it be a full-fledged DataGrid that replaces the current one?

**Comment by wieslawsoltes** [comment]
No plans yet on feature, we needed more performance then DataGrid and try to maintain compatibility as much as possible but if feature hinders the performance will not be included like usage of reflection. There are differences like no CollectionViewSource, no automatic column creation, sorting mechanism is different. The performance is most important.

**Comment by wieslawsoltes** [comment]
@kvalitetskontrollMarkaryd This is not just experimental control, it's already used in production (preview releases so far) so it's getting battle tested and for sure will get improvements and even new features if needed.

**Reply by EtherGhost** [reply]
Nice! Like I said, is looking good!
